src/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,41 @@
+package coffee.order;
+
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class CoffeeOrderBoard {
+
+  private int lastIndex = 0;
+  Queue<Order> orders = new LinkedList<>();
```

Comment:
what is the reason to use package private access modification?

Reply:
forgot))